MangoAPI.Client/src/app/components/chats/chats.component.ts

```diff
@@ -275,8 +329,13 @@ export class ChatsComponent implements OnInit, OnDestroy {
     }
 
     this.activeChatId = chatId;
-    this.activeChat = this.chats.filter((x) => x.chatId === this.activeChatId)[0];
+    const newActiveChat = this.chats.filter((x) => x.chatId === this.activeChatId)[0];
+    this.activeChat = newActiveChat;
+
+    this.activeChatBehaviorSubject.next(newActiveChat);
+
     this.getChatMessages(this.activeChatId);
+    this.scrollToEnd();
```

Comment:
Please fix method `this.scrollToEnd()` too, seems it scrolls not to exactly end of last message

Reply:
No, it scrolls to exactly end but it do it when pictures are not yet uploaded. When pictures become uploaded scrollbar goes up